### PR TITLE
Loggin error when applicationId is null - issue #379

### PR DESCRIPTION
**Pull request by aureamunoz**

### CODE REVIEW

**Comment by aureamunoz** [comment]
Can you please @cmoulliard review just the logs added, then I will work to fix the tested scenario.

File: .github/workflows/e2e-atomic-fruits-vault-crossplane.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-name: Temporary job to test all steps to install kind, vault & crossplane
+name: Job testing and end to end scenario with simple application. It tests all steps from kind, vault and crossplane installation.
```
Comment (cmoulliard) [inline .github/workflows/e2e-atomic-fruits-vault-crossplane.yml:1]: "Job testing and end to end scenario with simple application. It tests all steps from kind, vault and crossplane installation." -> "Job testing an end to end scenario with a simple Quarkus application. It tests all the steps starting from kind, vault and crossplane installation."

File: .github/workflows/e2e-atomic-fruits-vault.yml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-name: Temporary job to test all steps to install kind, vault & crossplane
+name: Job testing and end to end scenario with simple application. It tests all steps from kind and vault installation.
```
Comment (cmoulliard) [inline .github/workflows/e2e-atomic-fruits-vault.yml:1]: -> "Job testing an end to end scenario with a simple Quarkus application. It tests all the steps starting from kind and vault installation."

File: scripts/primaza.sh [hunk 3]
```diff
@@ -56,12 +56,11 @@ function primazaUsage() {
 function build() {
   cmdExec "mvn clean install -DskipTests -Dquarkus.container-image.build=true \
      -Dquarkus.container-image.push=true \
-     -Dquarkus.container-image.registry=${REGISTRY} \
+     -Dquarkus.container-image.registry=127.0.0.1:5000 \
```
Comment (cmoulliard) [inline scripts/primaza.sh:59]: Si le port défini pour les ENV VAR est de 5001: cfr `REGISTRY=${KIND_REGISTRY:-kind-registry:5001}` pour accéder au registre alors dle numéro du port pour faire un docker pull/push doit être aussi de 5001 ici !

Pq hard codé `-Dquarkus.container-image.registry=127.0.0.1:5000 ` alors qu'en variable ENV est définie `${REGISTRY} ` ?

File: scripts/primaza.sh [hunk 4]
```diff
@@ -56,7 +57,7 @@ function primazaUsage() {
 function build() {
   cmdExec "mvn clean install -DskipTests -Dquarkus.container-image.build=true \
      -Dquarkus.container-image.push=true \
-     -Dquarkus.container-image.registry=127.0.0.1:5000 \
+     -Dquarkus.container-image.registry=127.0.0.1:${REGISTRY_PORT} \
```
Comment (cmoulliard) [inline scripts/primaza.sh:60]: REGISTRY hostname should not be hard coded too `127.0.0.1` !

File: scripts/primaza.sh (hunk 3 above)
Comment (aureamunoz) [reply scripts/primaza.sh]: It's ok. I have changed and use the old kind-registry:5000. This was a leftover of several tests I was making...

File: scripts/primaza.sh (hunk 4 above)
Comment (aureamunoz) [reply scripts/primaza.sh]: It's ok. Look the last change and my previous comment

**Review by cmoulliard — APPROVED** [review]

**Pull request merged**